### PR TITLE
Feature/Shopping cart icon

### PR DESCRIPTION
**Pull request by VilleMiekkoja**
## Description

Add shopping cart icon to the core and react packages, and to documentation.

## Related Issue

https://helsinkisolutionoffice.atlassian.net/browse/HDS-971
https://helsinkisolutionoffice.atlassian.net/browse/HDS-972

## How Has This Been Tested?

Locally on developers machine
[👉 Storybook Demo](https://city-of-helsinki.github.io/hds-demo/shopping-cart-icon/?path=/story/icons-ui--iconshoppingcart)

## Screenshots (if appropriate):

image

### CODE REVIEW

File: packages/core/src/icons/ui/icon-shopping-cart.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@svg-load icon url(../../svg/ui/shopping-cart.svg);
+
+.hds-icon--heart {
```
Comment (laurakarhu) [inline packages/core/src/icons/ui/icon-shopping-cart.css:3]: Should it be .hds-icon--shopping-cart? Not heart?

**Review by laurakarhu — CHANGES_REQUESTED** [review]
One copy-paste error, otherwise looks good!

File: packages/core/src/icons/ui/icon-shopping-cart.css (hunk 1 above)
Comment (VilleMiekkoja) [reply packages/core/src/icons/ui/icon-shopping-cart.css:3]: - [ ] Fix copy-paste error

**Review by ronijaakkola — APPROVED** [review]

**Review by laurakarhu — APPROVED** [review]
Thanks!

**Pull request merged**